lib/tableau_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module TableauApi
-  VERSION = '2.0.0'.freeze
+  VERSION = '2.1.0'.freeze
```

Comment:
Does enough break in the tableau api itself that we should make this a major version?

Reply:
Oh hmm yeah, because of the site roles.